graphite_api/app.py

```diff
@@ -578,6 +589,8 @@ def tree_json(nodes, base_path, wildcards=False):
 
 
 def doImageRender(graphClass, graphOptions):
+    if not cairo:
```

Comment:
This shouldn't be silent. What about raising a custom exception here? There are guards in the code so we should never end up here, but for the future it would be nice to fail loudly.